packages/design-system/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -59,6 +59,24 @@ export const IconTrigger: Story = {
   },
 };
 
+export const idTest: Story = {
+  render: function Component(args) {
+    return (
+      <>
+        <p className="ds-u-margin--0 ds-u-display--inline">Tooltip with icon trigger</p>
+        <Tooltip {...(args as any)} />
+      </>
+    );
+  },
+  args: {
+    ariaLabel: 'Label describing the subject of the tooltip',
+    id: 'test-id-foo',
```

Comment:
created this story to quickly test user-created `id` values. storybook was occasionally breaking, citing some sort of async leak?